src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -113,6 +120,37 @@ def init_db():
 
     return db_conn
 
+def init_mgmt_interface_tables(db_conn):
+    """
+    Initializes interface maps for mgmt ports
+    :param db_conn: db connector
+    :return: tuple of mgmt name to oid map and mgmt name to alias map
+    """
+
+    db_conn.connect(APPL_DB)
+
+    mgmt_ports_keys = db_conn.keys(APPL_DB, mgmt_if_entry_table(b'*'))
+
+    mgmt_ports = [key.split(mgmt_if_entry_table(b''))[-1]
+        for key in (mgmt_ports_keys if mgmt_ports_keys is not None else [])]
+
+    if mgmt_ports:
+        oid_name_map = {get_index(mgmt_name): mgmt_name for mgmt_name in mgmt_ports}
+        logger.debug('Managment port map:\n' + pprint.pformat(oid_name_map, indent=2))
```

Comment:
>pprint [](start = 47, length = 6)

There is runtime overhead for pprint, even if the logger level is high.

Reply:
pprint is widely used in this file. Is this a problem? This function is called every reinit timeout (60 sec) Should I remove all usage of ppring?